### ISSUE TITLE
Using it on a binary without +x set results in a Permission Denied error when executing the static binary

### ISSUE THREAD

**Issue by Znuff** [JonathonReinhart/staticx#109]
Per title:
```
$ stat dynamic
  File: dynamic
  Size: 13005696        Blocks: 25408      IO Block: 4096   regular file
Device: 902h/2306d      Inode: 1050589     Links: 1
Access: (0664/-rw-rw-r--)  Uid: ( 1000/   )   Gid: ( 1000/   )
Access: 2020-01-10 21:21:06.618215958 +0100
Modify: 2020-01-10 21:13:16.478883211 +0100
Change: 2020-01-10 21:26:12.315189462 +0100
 Birth: -
$ ./.local/bin/staticx dynamic static
$ stat static
  File: static
  Size: 14034600        Blocks: 27416      IO Block: 4096   regular file
Device: 902h/2306d      Inode: 1048994     Links: 1
Access: (0775/-rwxrwxr-x)  Uid: ( 1000/   )   Gid: ( 1000/   )
Access: 2020-01-10 21:26:49.650820000 +0100
Modify: 2020-01-10 21:26:49.614820000 +0100
Change: 2020-01-10 21:26:49.666819922 +0100
 Birth: -
$ ./static
Failed to execv() /tmp/staticx-elFihB/.staticx.prog: Permission denied
```

Setting `+x` on the binary before, fixes this issue.

**Comment by JonathonReinhart** [comment]
Thanks for the issue! I find it a little strange using staticx to wrap a file that isn't already executable, but it would be a bit surprising to see the permission denied error, even after marking the *output* executable. I fixed this in #112.

**Comment by Znuff** [comment]
Thanks for the fix. My use case scenario was that I downloaded a binary from another server via http and I just run `staticx` directly on it, without doing `chmod +x` first. And the error message was a bit puzzling at first.

You are indeed correct that it's a bit strange.
